.circleci/config.yml

```diff
@@ -221,6 +218,7 @@ jobs:
       NX_E2E_CI_CACHE_KEY: e2e-circleci-macos
       NX_DAEMON: 'true'
       NX_PERF_LOGGING: 'false'
+      SELECTED_PM: 'npm' # use the previous runner
```

Comment:
Why is this `npm`?

Reply:
We run the command (on line 236) with `npx nx ...` so it automatically picks up `npm`. This just makes it explicit.

Reply:
We only run `macos` tests on `npm` (something to be changed for nightly) but not as a part of this PR